eslint.config.mts

```diff
@@ -37,6 +37,19 @@ export default tseslint.config(
       "@typescript-eslint/no-empty-object-type": "off",
       "@typescript-eslint/no-explicit-any": "warn",
       "@typescript-eslint/only-throw-error": "off",
+      "@typescript-eslint/consistent-type-definitions": ["error", "type"],
+      "@typescript-eslint/no-unused-vars": [
```

Comment:
I like having this rule around to mark unused things without eslint yelling at you, especially in array destructuring where the syntax would otherwise be `[, something,]`